index.html

```diff
@@ -73,6 +76,196 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
           </ul>
         </div>
       </section>
+
+      <!-- Work section -->
+      <section class="work">
+        <!-- work card 1 -->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio1.png" alt="Project 1" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Tonic</h1>
+            <ul class="work_info">
+              <li class="w_s">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 2 -->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio2.png" alt="Project 2" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Multi-Post Stories</h1>
```

Comment:
- [ ] For the semantic purpose only one h1 is allowed on the page. Kindly make sure you use another `header tag`
`Hint: You may consider using h2..6`